### PR TITLE
chore(deps): Update dependencies

### PR DESCRIPTION
**Pull request by ebbeknudsen**
## Description

D002 release:
- https://github.com/Energinet-DataHub/dh3-environments/actions/runs/12070800381

## Checklist
- [ ] Should the change be behind a feature flag?
- [ ] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
- [ ] Has it been considered whether data is being delivered to the wrong actor?
- [x] Subsystem test executed (dev_002/dev_003)
  - `dev_002`: https://github.com/Energinet-DataHub/dh3-environments/actions/runs/12115842036
- [ ] Is there time to monitor state of the release to Production?
- [ ] Reference to the task

### CODE REVIEW

File: source/B2BApi/B2BApi.csproj [hunk 1]
```diff
@@ -23,31 +23,31 @@ limitations under the License.
...
+        <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="8.2.1" />
+        <PackageReference Include="Microsoft.IdentityModel.Protocols" Version="8.2.1" />
+        <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="8.2.1" />
       <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.12.19">
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
       <PackageReference Include="System.Linq.Async" Version="6.0.1" />
-      <PackageReference Include="System.Text.Json" Version="8.0.5" />
+      <PackageReference Include="System.Text.Json" Version="9.0.0" />
```
Comment (dstenroejl) [inline source/B2BApi/B2BApi.csproj:50]: Are we sure this is OK? I mean for some libraries the 9.* would mean it's only compatible with .NET 9 and can cause issues with other versions.

File: source/SystemTests/SystemTests.csproj [hunk 2]
```diff
@@ -24,8 +24,8 @@
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
       <PackageReference Include="Nito.AsyncEx.Coordination" Version="5.1.2" />
-      <PackageReference Include="System.Text.Json" Version="8.0.5" />
-      <PackageReference Include="xunit" Version="2.9.0" />
+      <PackageReference Include="System.Text.Json" Version="9.0.0" />
```
Comment (dstenroejl) [inline source/SystemTests/SystemTests.csproj:27]: Same comment, are we sure we can bump to 9.*?

File: source/ApplyDBMigrationsApp/ApplyDBMigrationsApp.csproj [hunk 3]
```diff
@@ -29,7 +29,7 @@ limitations under the License.
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Text.Json" Version="9.0.0" />
```
Comment (dstenroejl) [inline source/ApplyDBMigrationsApp/ApplyDBMigrationsApp.csproj:32]: Are we sure it's OK to bump to 9.*?

**Review by dstenroejl — APPROVED** [review]
Approved, but left comments regarding System.Text 9.* - not sure if it's safe to bump that.

**Review by kerbou — APPROVED** [review]

**Pull request merged**